README.md

```diff
@@ -86,6 +86,17 @@ The DLRN instance architecture includes a secondary server, where all repos are
 ####`mock_tmpfs_enable`
 Enables the Mock TMPfs plugin. Note this will enable creation of a file system in RAM using up to 6 GB per worker, so be sure you have enough RAM and swap for all workers.
 
+####`server_type`
+Defines the server_type. It can be set to primary (default) or passive. Even when enabled in workers hiera configuration, passive servers will not enable:
+- cron jobs for run-dlrn.sh
+- rsync of new builds (they are suposed to be the destination of the synchronization)
+- Opening reviews in gerrit automatically on FTBFS
+- Sending mails on build failure
+Additionally, passive servers will redirect to buildlogs when users trying to access to current-passed-ci or current-tripleo
+
+####`enable_https`
+Enable https in apache configuration (requires proper certificates installed in the system). Default is true.
```

Comment:
imho, default should be false. We are requiring some specific certificates, which may or may not be available during the initial execution, so it is safest to not enable HTTPS by default.